_docs/services/external-domain-service.md

```diff
@@ -53,10 +53,17 @@ Name      | Required   | Description                   | Example
 
 ### `domain-with-cdn` plan
 
-Name      | Required   | Description                   | Example                           |
-----------|------------|-------------------------------|-----------------------------------|
-`domains` | *Required* | Your custom domain or domains | `my-domain.gov,www.my-domain.gov` |
+Name              | Required   | Description                   | Example                           |
+------------------|------------|-------------------------------|-----------------------------------|
+`domains`         | *Required* | Your custom domain or domains | `my-domain.gov,www.my-domain.gov` |
+`forward_cookies` | optional   | List of cookies to forward    | `"JSESSIONID,othercookiename"`    |
+
+#### forward_cookies option
 
+This option allows you to control what cookies to pass on to your application. By default, all cookies are passed.
+You can specify a list of cookie names (comma-separated) to whitelist. To pass no cookies, pass an empty string, e.g.
```

Comment:
s/whitelist/accept/ seems to be preferred terminology now.

Reply:
thanks for the reminder :)